hack/local-up-kubeedge.sh

```diff
@@ -21,6 +21,8 @@ LOG_LEVEL=${LOG_LEVEL:-2}
 TIMEOUT=${TIMEOUT:-60}s
 PROTOCOL=${PROTOCOL:-"WebSocket"}
 CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"remote"}
+CNI_DOWNLOAD_ADDR=${CNI_DOWNLOAD_ADDR}
+CNI_CONF_OVERWRITE=${CNI_CONF_OVERWRITE:-"false"}
```

Comment:
For users first use `local-up`, the overwrrite should be true by default?

Reply:
Okay, overwrite is now defaulting to true，added file backup and the overwrite reminder